platform/commonUI/inspect/src/gestures/InfoGesture.js

```diff
@@ -131,6 +126,13 @@ define(
             this.pendingBubble = this.$timeout(displayBubble, this.delay);
 
             this.element.on('mouseleave', this.hideBubbleCallback);
+
+            // Also make sure we dismiss bubble if representation is destroyed
+            // before the mouse actually leaves it
+            this.scopeOff =
+                this.element.scope() &&
```

Comment:
Representers (including GestureRepresenter) [do get skipped](https://github.com/nasa/openmctweb/blob/open195/platform/representation/src/MCTRepresentation.js#L182-L205) in these cases.

This handles a case where `scope()` returns undefined for reasons that are, well, unclear. The element has already re-entered the DOM at this point; I'm thinking that there's an interval between elements re-entering the DOM and being recognized by Angular (sufficient to provide answers to `scope()`) where this can occur.

I'm thinking this whole listener is actually unnecessary and should be removed. `MCTRepresentation` has the relevant scope already, is listening for `$destroy`, and is detaching all representers when this occurs.